.github/CODEOWNERS

```diff
@@ -178,6 +178,10 @@
 # ServiceLabel: %Cloud Shell
 # ServiceOwners:                                                   @maertendMSFT
 
+# ServiceLabel: %Code Signing
+# AzureSdkOwners:                                                  @jsquire
```

Comment:
```suggestion
```

The Azure SDK team does not own this package and will not be enhancing/maintaining it going forward, so we'll want just the service owners here.